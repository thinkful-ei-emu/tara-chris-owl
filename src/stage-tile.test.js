import React from 'react';
import ReactDOM from 'react-dom';
import StageTile from './stage-tile.js';
import renderer from 'react-test-renderer';

describe('StageTile Component', () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<StageTile name={'Tommy Thompson'} avatar={'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'} />, div);

        ReactDOM.unmountComponentAtNode(div);
  })

  it('renders the UI as expected', () => {
      const tree = renderer
        .create(<StageTile name={'Tommy Thompson'} avatar={'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'} />)
        .toJSON();
      expect(tree).toMatchSnapshot();  
      });
})