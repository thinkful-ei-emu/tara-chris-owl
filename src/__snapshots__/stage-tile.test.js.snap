// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StageTile Component renders the UI as expected 1`] = `
<div
  className="stage-tile"
>
  <h3>
    Tommy Thompson
  </h3>
  <button />
  <button />
  <img
    alt="Tommy Thompson's avatar"
    src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
  />
</div>
`;
